### DISCUSSION TITLE
What does Trakt Meta Correction do?

### DISCUSSION THREAD

**Discussion by ericbarnes88** [Goldenfreddy0703/Otaku#68] [Q&A] [answered]
I tried searching online for what it does. I searched for the trakt slugs as well, but what to use this for? I know trakt is bad at pulling anime and I'm assuming this is to help with this.

**Comment by Goldenfreddy0703** [comment, marked as answer]
Ok great question, back then we had very poor episode consistency so we basically relied of trakt for slugs, episode meta, ect. We have this trakt meta correction option incase that user would get the wrong episode meta from an anime. Example: Steins Gate and Steins Gate 0. That's why we made a context menu option called Trakt Meta Correction.

I'm gonna be honest, idk why we still have it and it stills create errors but we somehow still have it. 😂

We do still use trakt tho as a fallback for mappings.
